### ISSUE TITLE
Windows, testing: make //src/test/shell/... run on Windows

### ISSUE THREAD

**Issue by laszlocsomor** [bazelbuild/bazel#4930]
Blocked by:
- Bash runfiles library (https://github.com/bazelbuild/bazel/issues/4460)

Blocking:
- "Windows, testing: make all tests work on Window" (https://github.com/bazelbuild/bazel/issues/4292)

**Comment by laszlocsomor** [comment]
Update: I'm still working on this and P1 is appropriate.

**Comment by laszlocsomor** [comment]
Current status (at https://github.com/bazelbuild/bazel/tree/20637b7870a09a35b716f69a6d8ce5b4c5fe3722):
- `bazel query "tests(//src/test/shell/...)" | wc -l` --> 113
- `bazel query "tests(//src/test/shell/...)-tests(//src/test/shell:all_windows_tests)" | wc -l` --> 105

Currently only 7% of //src/test/shell/... runs on Windows.

**Comment by laszlocsomor** [comment]
Would be nice to have, but not critical, hence P3.

**Comment by meisterT** [comment]
The numbers from above comment have changed to
184 vs 115, so 37% run on Windows

**Comment by fmeum** [comment]
@bazelbuild/triage This is still relevant to improve the stability of existing features on Windows.
